src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const tableHead = document.querySelector('thead');
+const tableBody = document.querySelector('tbody');
+const rows = [...tableBody.rows];
+
+tableHead.addEventListener('click', (handler) => {
+  const header = handler.target.closest('th').innerText;
+
+  switch (header) {
+    case 'Name':
+      rows.sort((a, b) =>
+        (a.children[0].innerText).localeCompare(b.children[0].innerText));
+      break;
+
+    case 'Position':
+      rows.sort((a, b) =>
+        (a.children[1].innerText).localeCompare(b.children[1].innerText));
```

Comment:
combine it

Reply:
Could you please elaborate? I don't think I understand how to do it. Different indexes are used for Name and Position.